st2common/st2common/runners/__init__.py

```diff
@@ -0,0 +1,207 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Would you like this changed in this PR or a later one? I was just leaving it as it was for now, but happy to change it.